tests/upgrades/test_remoteexecution.py

```diff
@@ -96,17 +95,7 @@ def test_pre_scenario_remoteexecution_external_capsule(self, request, default_lo
             organization=[self.org.id],
             remote_execution_proxy=[entities.SmartProxy(id=2)],
         ).create()
-        client = VirtualMachine(
-            distro=DISTRO_RHEL7, provisioning_server=self.libvirt_vm, bridge=self.bridge
-        )
-        client.create()
-        request.addfinalizer(
-            lambda: cleanup_of_provisioned_server(
-                hostname=client.hostname,
-                provisioning_server=self.libvirt_vm,
-                distro=DISTRO_RHEL7,
-            )
-        )
+        client = rhel7_contenthost
```

Comment:
You can directly use it, you need not initialise the `client` variable for it.

Below line will change to:
```
rhel7_contenthost.install_capsule_katello_ca(capsule=self.proxy_name)
```
Same others.